client/src/lib/components/Cards/Card.tsx

```diff
@@ -97,51 +105,51 @@ export const Card: React.FC<CardProps> = ({
   hostPictureURL,
   hostName,
   pinned,
-}) => (
-  <>
-    <Wrapper onPress={onPress}>
-      <ContentWrapper>
-        <LeftCol>
-          <Header>
-            {title && <Display24 numberOfLines={2}>{title}</Display24>}
-            <Spacer4 />
-            <Byline
-              pictureURL={hostPictureURL}
-              name={hostName}
-              duration={duration}
-            />
-          </Header>
-        </LeftCol>
-        <GraphicsWrapper>
-          {lottie ? (
-            <Lottie source={lottie} autoPlay loop />
-          ) : image ? (
-            <Image resizeMode="contain" source={image} />
-          ) : null}
-        </GraphicsWrapper>
-      </ContentWrapper>
-      <Footer>
-        <LeftFooter>
-          {buttonText && (
-            <>
-              <CTAButton
-                LeftIcon={ButtonIcon}
-                small
-                variant="secondary"
-                onPress={onButtonPress ? onButtonPress : onPress}>
-                {buttonText}
-              </CTAButton>
-              <Spacer8 />
-            </>
-          )}
-        </LeftFooter>
-        <RightFooter>
-          {children}
-          <Interested active={pinned} onPress={onPinnedPress} />
-        </RightFooter>
-      </Footer>
-    </Wrapper>
-  </>
-);
+}) => {
+  return (
+    <>
+      <Wrapper onPress={onPress}>
+        <ContentWrapper>
+          <LeftCol>
+            <Tags tagIds={tagIds} duration={duration} />
```

Comment:
Me and Kajsa have tried to make the `Card` component very "stupid" and agnostic. Basically that `SessionCard` is the container that resolves all the data that `Card` need to render.
Here we resolve the tags in this component. Imo I would resolve the complete translated tags in `SessionCard` and just pass that as an array that you just render with `Tag` here.
The filtering logic of tags in `Tags` could be moved to a hook specific to `SessionCard` or as a more agnostic `useGetTagsByIds` in `content/`.